TriggerRedHatBuild.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Grab('io.github.http-builder-ng:http-builder-ng-core:1.0.4')
+
+import groovyx.net.http.HttpBuilder
+import groovyx.net.http.HttpException
+
+if (args.size() < 3) {
+  println 'Usage: groovy TriggerRedhatBuild.groovy <version> <projectId> <apiKey>'
```

Comment:
if it's likely you will add extra parameters to the script in the future it would be easier to add a proper arg parser instead of just using positional parameters.

Reply:
I think the parameters are set for now, but I'll keep CliBuilder in mind if I have to expand it.

Reply:
Rather than return here, shouldn't we throw an exception to ensure the underlying build fails?

Reply:
That's a great question.  I can definitely do that, but it appears that `runGroovy` runs an external groovy process. Will the build even recognize the exception coming out on stderr?  I'll go read the `runGroovy` code as well.

Reply:
the exception will cause the java process to exit with an error code an jenkins will notice that

Reply:
or just exit with an error code and not have the noise of a stackstrace

Reply:
👍 I did the System.err messages and then exit(1).

Reply:
Now that everything has a failure state  I had to check, I almost want to just do it with exceptions. :| leaving it for now, but I'll accept encouragement either way.